data/sles4sap/sdaf/SAP_SYSTEM.tfvars

```diff
@@ -107,7 +107,7 @@ network_logical_name = "%SDAF_VNET_CODE%"
 use_loadbalancers_for_standalone_deployments = false
 
 # use_private_endpoint is a boolean flag controlling if the key vaults and storage accounts have private endpoints
-use_private_endpoint = false
+use_private_endpoint = true
```

Comment:
Recommended by MS

Reply:
How to remember about this suggestion in the future? Where to document that we get this suggestion?

Reply:
It was a suggestion in sense "switch it to 'true'" during meeting while discussing other issue.
I would not really get too deep into this at this stage. Tfvars refining will be a topic by itself later.